src/replace.js

```diff
@@ -57,12 +57,23 @@ function asIdentifier(path, filename) {
       return asTag(targetPath, filename)
     }
     case 'CallExpression': {
-      return asFunction(targetPath, filename)
+      const isCallee = targetPath.get('callee') === path
+      if (isCallee) {
+        return asFunction(targetPath, filename)
+      } else {
+        return false
+      }
     }
     case 'JSXOpeningElement': {
       const jsxElement = targetPath.parentPath
       return asJSX(jsxElement, filename)
     }
+    case 'JSXClosingElement': {
+      // ignore the closing element
+      // but don't mark as unhandled (return false)
+      // we already handled the opening element
+      return true
```

Comment:
Previously this would hit the `default` case and return false which would trigger the invalid usage error.